<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fen_map</class>
 <widget class="QDialog" name="fen_map">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vue d'ensemble</string>
  </property>
  <widget class="QLabel" name="LA_Erreur">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>381</width>
     <height>201</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <layout class="QFormLayout" name="couche"/>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="text">
     <string>Lancer</string>
    </property>
   </widget>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>64</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>281</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
